src/interfaces.py

```diff
@@ -66,6 +66,13 @@ def get_positions(self) -> dict:
     def get_account_balance(self) -> float:
         pass
 
-    
+    @abc.abstractmethod
```

Comment:
These need to be added to both classes (AccountMT5 & AccountSimulator) if you intend to use them

Reply:
I see that you added these methods to AccountMT5 & AccountSimulator but they are not being called by the flask api currently

Reply:
Idk that he does need to add these to the simulator since the simulator is to test the backend and we have mocks on the frontend to mimic api calls that are not made.

Reply:
Ah. Shoot. No it's not. We're using a factory pattern and so the classes aren't independent. We could use `raise NotImplementedError` in the simulation classes for all of these.